WORKING-GROUP-PROCESSES.md

```diff
@@ -0,0 +1,142 @@
+
+* **Create a charter**. This should be a few paragraphs explaining:
+
+    * The mission of the working group
+
+    * The goals of the working group (problems being solved)
+
+    * The scope of the working group (topics, subsystems, code repos, areas of responsibility)
+
+* **Nominate an initial set of leads**. The leads set the agenda for the working group and serve as final arbiters on any technical decision. See [below](#heading=h.xodnjuc790rs)
+for information on the responsibilities of leads and requirements for nominating them.
+
+* **Prepare a Roadmap**. Create a preliminary 3 month roadmap for what the working group would focus on.
+
+* **Send an Email**. Write up an email with your charter, nominated leads, and roadmap, and send it to [istio-steering@](mailto:istio-steering@googlegroups.com). The steering working group will evaluate the request and decide whether the working group should be
+formed, whether it should be merely a subgroup of an existing working group, or whether it should be subsumed by an existing working group.
+
+## Setting up a working group
+
+Once approval has been granted by the steering group to form a working group, the working group leads need to take a few
+steps to establish the working group:
+
+* **Create a Google Drive Folder**. Create a folder to hold your working group documents within this parent
+[folder](https://drive.google.com/corp/drive/u/0/folders/0B7huSKYaiUN5LVdBeElXUGt3UGs). Call your folder <working group name>. 
+
+* **Create a Meeting Notes Document**. Create a blank document in the above folder and call it "<working group name> Group Meeting Notes".
+
+* **Create a Roadmap Document**. Create a document in the above folder and call it "<working group name> Group Roadmap". Put your initial
+roadmap in the document.
+
+* **Create a Wiki**. Create a wiki page on [GitHub](https://github.com/istio/istio) titled "<working group name> Design Decisions". This page
+will be used to track important design decisions made by the working group.
+
+* **Schedule a Recurring Meeting**. Create a recurring meeting (weekly or bi-weekly, 30 or 60 minutes) and call the meeting
+"<working group name> Group Sync-Up". Attach the meeting notes document to the calendar event. Generally schedule these meetings between
+9:00AM to 2:59PM Pacific Time. 
+
+* **Create a Public Google Group**. Call the group "istio-<working group name>" (all in lowercase, dashes for spaces). This mailing list
+must be open to all.
+
+* **Register the Working Group**. Go to [WORKING-GROUPS.md](https://github.com/istio/community/blob/master/WORKING-GROUPS.md) and
+add your working group name, the names of the leads, the working group charter, and a link to the meeting you created.
+
+* **Announce your Working Group**. Send a note to [istio-dev@](mailto:istio-dev@googlegroups.com),
+[istio-announce@](mailto:istio-announce@googlegroups.com), and [istio-steering@](mailto:istio-steering@googlegroups.com) to announce your new
```

Comment:
note: I tried to email istio-announce@ and it bounced back as it's restricted. so maybe that part can be left out and made part of the responsibility from someone who does have access to istio-announce@